csunplugged/topics/models.py

```diff
@@ -11,7 +11,7 @@ class GlossaryTerm(models.Model):
 
     #  Auto-incrementing 'id' field is automatically set by Django
     slug = models.SlugField(unique=True)
-    term = models.CharField(max_length=200, unique=True)
+    term = models.CharField(max_length=200)
```

Comment:
I agree, but because of the new creation technique, having `unique=True` fails in the first phase of creation as there are multiple `GlossaryTerm` objects with the term `""`.

However would having `term = models.CharField(max_length=200, unique=True, null=True)` solve this? I could test that out?